test/test_effect_classes.py

```diff
@@ -75,7 +88,9 @@ def test_start_loss():
     expect_effect(
         variant,
         transcript_id="ENST00000420316",
-        effect_class=StartLoss)
+        effect_class=StartLoss,
+        modifies_coding_sequence=True,
+        modifies_protein_sequence=True)
```

Comment:
I'm imagining any variant you would expect to give you a different protein (even if its sequence is unknown) has `modifies_protein_sequence` as True. The only weird case I haven't figured out yet is `ExonLoss`.